ai_genomics/getters/data_getters.py

```diff
@@ -0,0 +1,93 @@
+import os
+import pickle
+from fnmatch import fnmatch
+import boto3
+import json
+
+import pandas as pd
+
+from ai_genomics import bucket_name, logger, PROJECT_DIR
+from typing import Union, List
+from ai_genomics.utils.error_utils import Error
```

Comment:
There are some unused imports here.

```suggestion
import pickle
from ai_genomics import logger
from fnmatch import fnmatch
import boto3
import json
import pandas as pd
from typing import Union, List
from ai_genomics.utils.error_utils import Error
```